electrum/commands.py

```diff
@@ -1224,7 +1224,7 @@ def subparser_call(self, parser, namespace, values, option_string=None):
 def add_network_options(parser):
     parser.add_argument("-1", "--oneserver", action="store_true", dest="oneserver", default=None, help="connect to one server only")
     parser.add_argument("-s", "--server", dest="server", default=None, help="set server host:port:protocol, where protocol is either t (tcp) or s (ssl)")
-    parser.add_argument("-p", "--proxy", dest="proxy", default=None, help="set proxy [type:]host[:port], where type is socks4,socks5 or http")
+    parser.add_argument("-p", "--proxy", dest="proxy", default=None, help="set proxy [type:]host[:port] (or none to disable proxy), where type is socks4,socks5 or http")
```

Comment:
We should put it into quotes as otherwise it's not obvious what is meant here.
```suggestion
    parser.add_argument("-p", "--proxy", dest="proxy", default=None, help="set proxy [type:]host[:port] (or 'none' to disable proxy), where type is socks4,socks5 or http")
```

Reply:
Good point, thank you.